src/app/spartacus/fs-features.module.ts

```diff
@@ -1,28 +1,38 @@
 import { NgModule } from '@angular/core';
+import { AsmModule } from '@spartacus/asm';
+import { AsmOccModule } from '@spartacus/asm/occ';
+import { CartPageEventModule } from "@spartacus/cart/base/core";
+import { CheckoutModule } from "@spartacus/checkout/base";
+import { CheckoutOccModule } from "@spartacus/checkout/base/occ";
+// check TODO:Spartacus - // TODO:Spartacus - Class CartModule has been removed and is no longer part of the public API. The cart base feature is now extracted to a lazy loadable library @spartacus/cart/base.  See the release documentation for more information.  While it's not identical, the new module 'CartBaseCoreModule' in '@spartacus/cart/base/core' is the closest equivalent in the new cart library.
```

Comment:
Should this comments about removed Modules stay?

Reply:
removed